Include/internal/pycore_object.h

```diff
@@ -178,7 +178,7 @@ _Py_DECREF_SPECIALIZED(PyObject *op, const destructor destruct)
     }
     _Py_DECREF_STAT_INC();
 #ifdef Py_REF_DEBUG
-    _Py_DEC_REFTOTAL(_PyInterpreterState_GET());
+    _Py_DEC_REFTOTAL(PyInterpreterState_Get());
```

Comment:
Because the macro version expands to a load of a thread-local variable, which results in disgusting relocations in the JIT's code. :)

I feel comfortable doing this since it's only there on debug builds.